dask/dataframe/io.py

```diff
@@ -606,6 +605,15 @@ def read_hdf(path_or_buf, key, start=0, stop=None, columns=None,
     return DataFrame(dsk, name, columns, divisions)
 
 
+@wraps(pd.read_hdf)
+def read_hdf(pattern, key, start=0, stop=None, columns=None,
+             chunksize=1000000):
+    paths = sorted(glob(pattern))
+    reader = lambda p: _read_single_hdf(p, key, start=start, stop=stop,
+            columns=columns, chunksize=chunksize)
```

Comment:
Could we avoid the lambda here and use the function call directly in the next line?